package.json

```diff
@@ -23,13 +22,12 @@
     ]
   },
   "scripts": {
-    "prebuild": "npm run clean",
+    "prebuild": "node -e \"var s=require('shelljs'),d=['generators','es5-check','es5-test'];s.rm('-rf',d);s.mkdir('-p',d)\"",
```

Comment:
The other thing here is that I fully intend a future release to change these `node -e` scripts to something much cleaner, but I haven't had time to write a PR for shelljs or to write a CLI wrapper package.